### PR TITLE
flutter: add page

### PR DESCRIPTION
**Pull request by christopherkuemmel**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

**Comment by christopherkuemmel** [comment]
`flutter emulators` could be a possibility, listing all available emulators. Followed by a `flutter emulators --launch <emulator id>` to launch a specific one. Or `flutter run` to deploy your code to a running device. However, these commands are specific to the emulators or paths of the project. And to keep it simple for **tldr** I wanted to leave it at the simple commands. 

`flutter help` will list all available commands.

**Comment by agnivade** [comment]
Got it.

File: pages/common/flutter.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+- Call general help:
+
+`flutter help`
+
+- Call help on a command:
+
+`flutter help {{command}}`
+
+- Execute Flutter command:
```
Comment (sbrl) [inline pages/common/flutter.md]: Perhaps `Execute a Flutter command` would read better?

File: pages/common/flutter.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# flutter
+
+> Google’s free and open source cross-platform mobile app SDK.
+
+- Check the Flutter version:
+
+`flutter --version`
+
+- Call general help:
```
Comment (sbrl) [inline pages/common/flutter.md]: Perhaps `Display general help` would read better?

File: pages/common/flutter.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+`flutter help`
+
+- Call help on a command:
```
Comment (sbrl) [inline pages/common/flutter.md]: Perhaps `Display help about a specific command` would read better?

File: pages/common/flutter.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+# flutter
+
+> Google’s free and open source cross-platform mobile app SDK.
```
Comment (sbrl) [inline pages/common/flutter.md]: Perhaps ...`free, open source, and cross-platform`.... would read better?

File: pages/common/flutter.md (hunk 1 above)
Comment (christopherkuemmel) [reply pages/common/flutter.md]: That's right, that sounds better.

File: pages/common/flutter.md (hunk 2 above)
Comment (christopherkuemmel) [reply pages/common/flutter.md]: That's right, that sounds better.

File: pages/common/flutter.md (hunk 3 above)
Comment (christopherkuemmel) [reply pages/common/flutter.md]: That's right, that sounds better.

File: pages/common/flutter.md (hunk 4 above)
Comment (christopherkuemmel) [reply pages/common/flutter.md]: That's right, that sounds better.

**Pull request merged**